src/js/jsx/sections/style/Fill.jsx

```diff
@@ -70,6 +71,7 @@ define(function (require, exports, module) {
             this.getFlux().actions.shapes
                 .setFillColorThrottled(this.props.document, this.props.layers, color,
                     { coalesce: coalesce });
+            headlights.logEvent("edit", "color-input", "color-change");
```

Comment:
Also note that the _opacityChanged handler is called by the color picker as well.